modern/.env

```diff
@@ -25,7 +25,7 @@ CLANG12_VERSION=12.0.0
 CLANG13_VERSION=13.0.0
 CLANG14_VERSION=14.0.0
 
-DISTRO=ubuntu16.04
-DISTRO_VERSION=16.04
+DISTRO=ubuntu18.04
```

Comment:
Please, revert it. Ubuntu 16.04 should be used as main version. The 18.04 is not used by ConanCenterIndex. In the future we should move to 20.04 or 22.04 directly, so we will skip 18.04.

Reply:
2 things on this:

1. I thought according to this https://github.com/conan-io/conan-docker-tools/tree/master/modern#ubuntu-1804-bionic-as-default-distro-october-2022 that  Ubuntu 18.04 was he base distro.  Is that incorrect?
2. Is the .env file used in the Jenkins process that generates images for ConanCenterIndex?  I would think not since the CONAN_VERSION is 2.0.17 and it seems you are still using 1.0 for central.

Reply:
> I thought according to this [master/modern#ubuntu-1804-bionic-as-default-distro-october-2022](https://github.com/conan-io/conan-docker-tools/tree/master/modern?rgh-link-date=2024-01-25T13%3A20%3A25Z#ubuntu-1804-bionic-as-default-distro-october-2022) that Ubuntu 18.04 was he base distro. Is that incorrect?

Yes, that file is incorrect. When we moved to 18.04 we actually had a problem with packages already available in Conan Center, built with 16.04. The problem is glibc version, each Ubuntu version has a newer version. So, you can use an older in glibc version in a newer distro, but not the opposite. It resulted in a breakage in Conan Center, so we reverted to 16.04. That document is wrong and need an update.

> Is the .env file used in the Jenkins process that generates images for ConanCenterIndex? I would think not since the CONAN_VERSION is 2.0.17 and it seems you are still using 1.0 for central.

Jenkins uses .env partially, the CONAN_VERSION it consumes directly from build parameters, like, when a new Conan client is released, the CI triggers another job to Jenkins build the docker images, it passes the Conan version released as well. You take a look in the `.ci` folder, it contains all Jenkinsfiles used.